jobs/build/custom/Jenkinsfile

```
@@ -113,7 +129,7 @@ node {
                 if (rpms.toUpperCase() != "NONE") {
                     currentBuild.displayName += rpms.contains(",") ? " [RPMs]" : " [${rpms} RPM]"
                     currentBuild.description = "building RPM(s): ${rpms}\n"
-                    command = "--working-dir ${doozer_working} --group 'openshift-${params.BUILD_VERSION}' "
+                    command = "--working-dir ${doozer_working} --data-path ${doozer_data_path} --group 'openshift-${params.BUILD_VERSION}' "
```

hmm, I realize it was this way when you found it, but the longer it gets, the more obvious... why isn't this just:
```suggestion
                    command = doozerOpts
```

and the same for the next couple changes